### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by ThunderSon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# WWW ZAP Project
+
+The new home for the ZAP project which can be found on <https://www2.owasp.org/www-project-zap/>.
```
Comment (thc202) [inline README.md]: `The OWASP home...`? Would avoid the need to remove the "new" later also emphasise OWASP as we plan to have another home/site.

**Comment by thc202** [comment]
Would you mind excluding the file, IMO it's not necessary to deploy it.

**Comment by ThunderSon** [comment]
I believe it's better to have more references. It's a little faster as well to jump from here to the website, just like the website links to GitHub. And for the new word, I definitely can fix that. Let me know what you think

**Comment by kingthorin** [comment]
Updated. I think this should be good to go now.

**Comment by thc202** [comment]
Thank you (both)!

**Pull request merged**